lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -34,6 +34,11 @@ FAILED_RFC:
     next:
       targetState: NO_MATCH_PAGE_RFC
 
+FAILED_UPDATE_ADDRESS:
+  events:
+    next:
+      targetState: NO_MATCH_PAGE_ADDRESS
```

Comment:
I think there's probably an oversight here for the RFC as well - these states should all go via TICF as well, when the feature flag is enabled.

It's also worth checking whether we need a separate failure state for this? In the figma it's the same as should be used for repeat fraud check, so I think we can just direct to that page instead.

(We might need a separate ticket to change the design of that page, if the current one needs updating)

Reply:
@Joe-Edwards-GDS thanks, yes they are the same, I'm a bit reluctant to use the FAILED_RFC **and** change the RFC to go via TICF at the same time as the RFC part would not be tested as part of this ticket. Might be worth a separate ticket?

Reply:
Yeah - although we clearly don't have a test for it. If you prefer I'll raise a bug ticket for the RFC case

Reply:
@Joe-Edwards-GDS I've added the TICF check and removed a bit of replication in the states, should be easy enough to  do that ticket when the time comes and get them both using the same error state